lib/internal/modules/esm/get_source.js

```diff
@@ -33,8 +34,8 @@ async function defaultGetSource(url, { format } = {}, defaultGetSource) {
   } else {
     throw new ERR_INVALID_URL_SCHEME(['file', 'data']);
   }
-  if (manifest) {
-    manifest.assertIntegrity(parsed, source);
+  if (policy && policy.manifest) {
```

Comment:
Can this possibly be simplified to `policy?.manifest` now?

Reply:
i guess it could, won't land on 12.x but that seems fine

Reply:
fixed